### PR TITLE
Added feature comparison table

### PR DESCRIPTION
**Pull request by JustinReynolds-MM**

### CODE REVIEW

**Comment by amyblais** [comment]
I made a few tweaks, though it doesn't reflect here ^

**Comment by lindentea** [comment]
@amyblais - was Eric the one who helped draft this? (Just wondering if it would be better for him to review, or if he was already involved in the initial draft).

**Comment by lfbrock** [comment]
@amyblais, was Eric the one who helped draft this? (Just wondering if it would be better for him to review, or if he was already involved in the initial draft).

**Comment by amyblais** [comment]
I have no idea who drafted this, @JustinReynolds-MM do you know?

**Comment by JustinReynolds-MM** [comment]
Jason sent me the doc with the information in it, maybe him?

File: source/overview/faq.rst [hunk 1]
```diff
@@ -31,6 +31,79 @@ We test to 60,000 concurrent users regularly, and have a peak concurrent utiliza
...
 
+What features are available on different Mattermost clients?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following chart highlights the end user features of Mattermost and their support across Web, Desktop and Mobile applications (iOS and Android).
+
+.. csv-table::
+    :header: "Feature", "Web", "Desktop", "Mobile"
+
+    "**Channels**"
```
Comment (jasonblais) [inline source/overview/faq.rst]: In addition to bolding, propose also capitalizing so the section headers are more clear.

File: source/overview/faq.rst [hunk 2]
```diff
@@ -31,6 +31,79 @@ We test to 60,000 concurrent users regularly, and have a peak concurrent utiliza
 
 Mattermost provides an open source, well-documented load test simulator to verify that your Mattermost deployment can achieve the stated scale benchmarks ahead of production deployment. 
 
+What features are available on different Mattermost clients?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following chart highlights the end user features of Mattermost and their support across Web, Desktop and Mobile applications (iOS and Android).
```
Comment (jasonblais) [inline source/overview/faq.rst]: `...Desktop (Windows, MacOS and Linux)...`

File: source/overview/faq.rst [hunk 3]
```diff
@@ -31,6 +31,79 @@ We test to 60,000 concurrent users regularly, and have a peak concurrent utiliza
...
+    "(Windows) Add Mattermost to Start menu, taskbar and system tray", " ", "X", 
+    "(Windows/Mac) Deep link to the desktop app via mattermost:// protocol if app is already installed", " ", "X", 
+    "(Mac) Add Mattermost to the applications Dock", " ", "X", 
+    "(Linux) Desktop Entry for the application to more easily integrate into a desktop environment", " ", "X", 
+
+**Notes:**
+
+* X - Full Support
+
+* S - Support for ‘collapse’, ‘expand’, ‘join’, ‘open’, ‘leave’, ‘logout’, ‘msg’, ‘grpmsg’, ‘rename’, ‘invite_people’, ‘remove’ not yet included
```
Comment (jasonblais) [inline source/overview/faq.rst]: I'm curious - is "S" in this case equivalent to "O" -- given O implies partial support.

File: source/overview/faq.rst [hunk 4]
```diff
@@ -31,6 +31,79 @@ We test to 60,000 concurrent users regularly, and have a peak concurrent utiliza
...
+    "Account settings", "X", "X", "O"
+    "OAuth login with Google/Office 365", "X", "X",
+    "Connect to multiple Mattermost servers from a single interface and switch with shortcut keys (similar to tabs in web browser", " ", "X", 
+    "Auto-start Mattermost when a user logs into their machine", " ", "X"    
+    "(Windows) Add Mattermost to Start menu, taskbar and system tray", " ", "X", 
+    "(Windows/Mac) Deep link to the desktop app via mattermost:// protocol if app is already installed", " ", "X", 
+    "(Mac) Add Mattermost to the applications Dock", " ", "X", 
+    "(Linux) Desktop Entry for the application to more easily integrate into a desktop environment", " ", "X", 
+
+**Notes:**
```
Comment (jasonblais) [inline source/overview/faq.rst]: `Notes` --> `Legend`

File: source/overview/faq.rst [hunk 5]
```diff
@@ -31,6 +31,79 @@ We test to 60,000 concurrent users regularly, and have a peak concurrent utiliza
 
 Mattermost provides an open source, well-documented load test simulator to verify that your Mattermost deployment can achieve the stated scale benchmarks ahead of production deployment. 
 
+What features are available on different Mattermost clients?
```
Comment (jasonblais) [inline source/overview/faq.rst]: Propose adding this question to a new "Product Questions" section, and moving below "Use Cases".

**Pull request merged**